Analyses/test/geom_bfieldplotter.txt

```diff
@@ -0,0 +1,26 @@
+// -*- mode:c++ -*-
+
+// default to the most current implementation, since geometry shouldn't matter too much other than running
+#include "Mu2eG4/geom/geom_common_current.txt"
+
+//redefine the fields here if desired, to plot different maps
+vector<string> bfield.innerMaps = {
+  "BFieldMaps/Mau13/DSMap.header"
+ ,"BFieldMaps/Mau13/PSMap.header"
+ ,"BFieldMaps/Mau13/TSuMap_fix.header"
+ ,"BFieldMaps/Mau13/TSdMap.header"
+ ,"BFieldMaps/Mau13/PStoDumpAreaMap.header"
+ ,"BFieldMaps/Mau13/ProtonDumpAreaMap.header"
+ ,"BFieldMaps/Mau13/DSSweepExtension.header"
+};
+//DSSweepExtension supersedes this
+ // ,"BFieldMaps/Mau13/DSExtension.header"
```

Comment:
My recommendation is to get rid of comments that capture transient state.  git is good at documenting transient state.

Reply:
Good call, I've removed these comments